package.json

```diff
@@ -18,7 +18,7 @@
   "main": "app/index.js",
   "repository": "h5bp/generator-server-configs",
   "scripts": {
-    "test": "mocha --timeout 50000"
+    "test": "mocha --timeout 60000"
```

Comment:
PS: testing it revealed that it doesn't work on `iojs`, which is a good thing to know.